apps/client/src/components/Conclusion/ConclusionOutcome.tsx

```diff
@@ -44,31 +40,14 @@ export const ConclusionOutcome: React.FC<Props> = ({
     window.open(Olo.home, "_blank");
   };
 
-  return (
-    <>
-      <Paragraph gutterBottom={isMobile ? 16 : 20}>
-        U bent klaar met de vergunningcheck. Dit is de uitkomst:
-      </Paragraph>
-      <ComponentWrapper marginBottom={16}>
-        <Heading forwardedAs="h2">
-          {contactConclusion?.title || heading}.
-        </Heading>
-      </ComponentWrapper>
-      {needsPermit && (
-        <Paragraph gutterBottom={16}>
-          U kunt deze vergunning aanvragen bij het landelijk omgevingsloket
-        </Paragraph>
-      )}
-      {contactConclusion?.description && (
-        <div data-testid={NEED_CONTACT}>
-          <Markdown
-            eventLocation={sections.CONCLUSION}
-            source={contactConclusion.description}
-          />
-        </div>
-      )}
-      <HideForPrint>
-        {needsPermit && (
+  const conclusionText = {
+    needPermit: {
+      title: "U hebt een omgevingsvergunning nodig.",
+      contentParagraph: (
+        <>
+          <Paragraph gutterBottom={16}>
+            U kunt deze vergunning aanvragen bij het landelijk omgevingsloket
+          </Paragraph>
```

Comment:
Shall we put this part in a separate component to make the object more understandable?